css-structure/css-structure_custom-properties/css/styles.css

```diff
@@ -75,25 +75,34 @@ a {
   display: inline-block;
   font-size: smaller;
   padding: 8px;
-  border-radius: 4px;
-  background-color: hsl(263, 100%, 61%);
-  color: #fff;
+  border-radius: var(--border-radius-small);
+  background-color: var(--color-primary-color);
+  color: var(--color-white-color);
 }
 
 a:hover,
 a:focus-visible {
-  background-color: rgb(48 48 48);
+  background-color: var(--color-black-light-color);
   text-decoration: none;
 }
 
 mark {
-  background-color: rgb(130, 54, 255);
-  color: #ffffff;
-  border-radius: 4px;
-  letter-spacing: 0.025em;
+  background-color: var(--color-primary-color);
+  color: var(--color-white-color);
+  border-radius: var(--border-radius-small);
+  letter-spacing: var(--letter-spacing-size);
 }
 
 ::selection {
-  background-color: #303030;
-  color: #8236ff;
-}
\ No newline at end of file
+  background-color: var(--color-black-light-color);
+  color: var(--color-primary-color);
+}
+
+:root {
+  --color-black-light-color: rgb(48, 48, 48);
+  --color-primary-color: rgb(130, 54, 255);
+  --color-white-color: rgb(255, 255, 255);
+  --border-radius-small: 4px;
+  --letter-spacing-size: 0.025em;
+  --margin-null: 0;
```

Comment:
Maybe the name of margin-null could be different, because of the case that you want to change that margin to something different than 0 (for example --margin-of-text)